### PR TITLE
feat(env): local dev compatible with GitHub Workspace

### PR DESCRIPTION
**Pull request by robertu7**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ try {
 const PORT = 4000
 const app = express()
 const httpServer = http.createServer(app)
+app.set('trust proxy', ['loopback', 'linklocal', 'uniquelocal'])
```
Comment (devformatters) [inline src/index.ts:34]: 🤔

**Review by devformatters — APPROVED** [review]
Go.

File: src/index.ts (hunk 1 above)
Comment (robertu7) [reply src/index.ts:34]: In GitHub Workspace with port forwarding, `req.hostname` of API requests is `localhost` when calling `https://robertu7-thematters-matters-server-hj5h-4000.githubpreview.dev/graphql`, and which isn't what we expected. 

related doc: http://expressjs.com/en/guide/behind-proxies.html

Comment (robertu7) [reply src/index.ts:34]: These changes need a test on local and develop environments.

**Pull request merged**